packages/graphql-tool-utilities/README.md

```diff
@@ -16,51 +16,7 @@ yarn add graphql-tool-utilities
 
 ## Usage
 
-### Configuration
```

Comment:
should i add a mention in here for where these functions now live?

Reply:
I don't think you need to worry about it